sdk/resourcemanager/azure-resourcemanager-samples/src/test/java/com/azure/resourcemanager/samples/ComputeSampleTests.java

```diff
@@ -195,6 +195,7 @@ public void testCreateVirtualMachineEncryptedUsingCustomerManagedKey() {
     }
 
     @Test
+    @DoNotRecord(skipInPlayback = true)
     public void testCloneVirtualMachineToNewRegion() {
         Assertions.assertTrue(CloneVirtualMachineToNewRegion.runSample(azureResourceManager));
     }
```

Comment:
@XiaofeiCao Please check why PLAYBACK is slow. Should use `ResourceManagerUtils.getDelayDuration` for any duration in code.

Reply:
@XiaofeiCao Also please check if JUnit can have a global setting for timeout for each test case (e.g. set it to 5 sec per test, enabled only at PLAYBACK).

Reply:
Got it, The issue is that I did not use the `ResourceManagerUtils.getDelayDuration` in `waitForCopyStartCompletion`. 

Sorry for that.. Will look into it.